### PR TITLE
prep-javascript-operators-and-expressions

### PR DESCRIPTION
**Pull request by Ortiz-Andrew**
Screenshot 2023-06-21 at 10 33 15 PM

### CODE REVIEW

File: prep-javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const fullName = `${firstName}${lastName}`;
+console.log(fullName);
+console.log(typeof fullName);
+const pH = 7;
+const isAcidic = pH < 5;
+console.log(isAcidic);
+console.log(typeof isAcidic);
+const numberOfSoldiers = 300;
+const isSparta = 1;
+console.log(numberOfSoldiers === isSparta);
```
Comment (BrettAlbright) [inline prep-javascript-operators-and-expressions/main.js]: you want to save the result of comparing the number of soldiers to 300 into the `isSparta` variable and then log that variable

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Comment by Ortiz-Andrew** [comment]
Screenshot 2023-06-22 at 1 57 36 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**